### PR TITLE
polish sui-tool result display and add latency 

### PR DESCRIPTION
**Pull request by longbowlu**
1. for fetch-transaction and fetch-object in concise mode, group by results to make results more readable
2. measure and print perceived latency for every validator grpc call

Example:
```
./sui-tool fetch-object --id  0xdeef067fe8ac29e112dbba3145a51a0e93f423b2 --genesis genesis.blob --version 3 --history --verbosity concise --concise-no-header
```

```
./sui-tool fetch-object --id  0xdeef067fe8ac29e112dbba3145a51a0e93f423b2 --genesis genesis.blob --version 76
seq num: SequenceNumber(76)
obj ref: o#3c3a0c3036287fad33d0a6493e6cfa4ea7fea04f7dc9123e92cd287219146224
parent cert: cZMHDxg6vuYlaWDyPj4kMb8/6Vw8bw0xWvQhpyUWx/8=
owner: Account Address ( 0xf676b4648ccfed31e882ff766cd51c7d86d0a955 )
lock: None
        0    k#8347c6e53b00c3d9c2da8e84825f8d02e8b404db22d9a270c1e0830b08782c9ec215ab8dcd156d7858071569b2e4403c01a4a38c7e694f92991261fe95220f843a9d4ff8e9a60900d01b898fd1a016d83fcca6d6f8354ba217d2568e1b286482 /dns/validator-3.devnet.sui.io/tcp/8080/http             (using 0.254 seconds)
        1    k#9839edf49421adadc8f863330ab1e2ac665446dfd30304c17082063a0de7a8fea6969f059a618f88f32251c21f16916a06c2dbc80ba2a1625fc6d77d1ab9f380c7801a9d3069df669897547cea24de92f6a8a44aa4d68f81f5d48ce10a618317 /dns/validator-0.devnet.sui.io/tcp/8080/http             (using 0.250 seconds)
        2    k#a85762f23eda30cf5f7f28a45736a3fab5b9b18d3d276f56b426607645759f3e52d67ba5525427fb71a8efb814854bfc113a04850ce63ac297110d45db459caa64e14e58bd4e42acc34aab8623d4e3501a9b98cd7b8c077db0d0d8024924c1fa /dns/validator-1.devnet.sui.io/tcp/8080/http             (using 0.249 seconds)
        3    k#a8cde123c0c6065ebf929d1bfc85baa69318c7282b70f54bceea046a96aa57e31b37140808a7157345ab313a44794ba1093317a3816d29fae3d71d8f40a71b3681ca63df3fd0df59f5a98ec15c9a9b976f3ee327c018b2e525a2c9586b8ca4f7 /dns/validator-2.devnet.sui.io/tcp/8080/http             (using 0.268 seconds)
```

### CODE REVIEW

File: crates/sui-tool/src/commands.rs [hunk 1]
```diff
@@ -53,30 +56,24 @@ pub enum ToolCommand {
...
+        /// Concise mode groups responses by results.
+        /// prints tabular output suitable for processing with unix tools. For
         /// instance, to quickly check that all validators agree on the history of an object:
         ///
         ///     $ sui-tool fetch-object --id 0x260efde76ebccf57f4c5e951157f5c361cde822c \
         ///         --genesis $HOME/.sui/sui_config/genesis.blob \
-        ///         --history --concise   --no-header \
-        ///         | sort -k2 -k1 \
-        ///         | uniq -f1 -c
+        ///         --history --concise
```
Comment (mystenmark) [inline crates/sui-tool/src/commands.rs:65]: I actually use that `sort | uniq` incantation and I would like to keep being able to use it - the entire reason I added the `--concise` mode was to be able to process the output with unix tools. this change breaks that. I definitely agree that the new output is prettier when viewed as a whole, however.

Can you simply add a third output mode? Perhaps make your new output the default, and make the current default output triggered with `--verbose`?

**Review by mystenmark — CHANGES_REQUESTED** [review]

**Comment by longbowlu** [comment]
will look to resume this work

File: crates/sui-tool/src/commands.rs (hunk 1 above)
Comment (longbowlu) [reply crates/sui-tool/src/commands.rs]: PR updated with `verbosity` param replacing `concise`. It supports 3 modes "groupped" (default), "concise" and "verbose"

**Review by mystenmark — APPROVED** [review]
Please remove the genesis.blob file, LGTM after that

**Pull request merged**